gis/geoexpress-sdk/Portfile

```diff
@@ -2,25 +2,41 @@ PortSystem          1.0
 
 name                geoexpress-sdk
 version             9.5.4.4709
+revision            1
 maintainers         {vince @Veence}
 platforms           darwin
 description         LizardTech's SDK for reading MrSID and JPEG 2000 imagery
 long_description    LizardTech's SDK for reading MrSID and JPEG 2000 imagery. This \
                     SDK is NOT open source software, and must be downloaded \
-                    manually from http://www.lizardtech.com/developer/ \
+                    manually from https://www.extensis.com/support/developers \
                     Please carefully review the SDK agreement prior to downloading \
                     and installing this software.
-homepage            http://www.lizardtech.com/developer/
-master_sites        http://www.lizardtech.com/developer/
+homepage            https://www.extensis.com/support/developers
+master_sites        https://bin.extensis.com/download/developer/
 categories          gis
 license             Restrictive NoMirror
 
-distname            MrSID_DSDK-${version}-darwin16.universal.clang80
-worksrcdir          MrSID_DSDK-${version}-darwin16.universal.clang80
+if { ${os.major} >= 16 } {
+    distname        MrSID_DSDK-${version}-darwin16.universal.clang80
+    worksrcdir      MrSID_DSDK-${version}-darwin16.universal.clang80
+    checksums       rmd160  1afa1c4bf109712cfbc436c20431cf102ca7a175 \
+                    sha256  086f5dd508484e7ad915fb76ab962e96099b37ca8d759a0112d51eaa007cb488 \
+                    size    56829152
 
-checksums           rmd160  62c164015c1a08ea5ad8509a6d6734da0095a357 \
-                    sha256  2dae719de7e852ac2aa46c0de5781924350ec84bf2c0c4915a3eab2f0e8301d2 \
-                    size    56829205
+    # Suspected stealth update: https://trac.macports.org/ticket/56535
+    # Remove this when version is updated
+    dist_subdir   ${name}/${version}_1
```

Comment:
Actually, since MacPorts requires the user to manually download the distfile, I don't think there's a point in requiring it to be placed in a different location. I will remove these line.

Reply:
I will also remove the revision increment. I'm not certain it should be mandatory for existing users in this case.